frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -425,19 +468,43 @@ function AnnotationsCsvInput({
           name: annotation.dataSetName,
         });
 
-        const volumeTracings: ServerVolumeTracing[] = await Promise.all(
+        const volumeServerTracings: ServerVolumeTracing[] = await Promise.all(
           annotation.annotationLayers
-            .filter((layer) => layer.typ === "Skeleton")
+            .filter((layer) => layer.typ === "Volume")
             .map(
               (layer) =>
                 getTracingForAnnotationType(annotation, layer) as Promise<ServerVolumeTracing>,
             ),
         );
+        // TODO: make bboxs a member again
+        const volumeTracings = volumeServerTracings.map((tracing) =>
+          serverVolumeToClientVolumeTracing(tracing),
+        );
+        let userBoundingBoxes = volumeTracings[0]?.userBoundingBoxes;
+        if (!userBoundingBoxes) {
+          const skeletonLayer = annotation.annotationLayers.find(
+            (layer) => layer.typ === "Skeleton",
+          );
+          if (skeletonLayer) {
+            const skeletonTracing = await getTracingForAnnotationType(annotation, skeletonLayer);
+            userBoundingBoxes = convertUserBoundingBoxesFromServerToFrontend(
+              skeletonTracing.userBoundingBoxes,
+            );
+          } else {
+            new Error(`Annotation ${annotation.id} has neither a volume nor a skeleton layer`);
+          }
+        }
+
+        console.log(volumeTracings); // TODOM remove
```

Comment:
TODO

Reply:
thanks, sorry I sometimes tend to skip reading comments :see_no_evil: